Kitodo/src/main/java/de/sub/goobi/metadaten/copier/MetadataPathSelector.java

```diff
@@ -252,7 +252,7 @@ protected String findIn(DocStructInterface supernode) {
      */
     public int getIndex() {
         if (index instanceof Integer) {
-            int a = ((Integer) index).intValue();
+            int a = (Integer) index;
```

Comment:
`Integer` class can contain `null` value but primitive not. You may replace `((Integer) index).intValue()` with `index.intValue()` to archive same behaviour?

Reply:
Basically it is the same, doesn't matter if you use index or index.intValue:
https://stackoverflow.com/questions/13914317/assigning-a-null-value-to-an-int

For this case important part is also instance of - it returns false if index is null so assign will not be performed.